helm/cmp-platform/requirements.lock

```diff
@@ -0,0 +1,24 @@
+dependencies:
+- name: openam
+  repository: file://../openam
+  version: 6.0.0
```

Comment:
Just verifying that 6.0.0 is the correct version here and elsewhere in this file.